code/modules/admin/admin.dm

```diff
@@ -10,6 +10,15 @@
 
 
 ///////////////////////////////////////////////////////////////////////////////////////////////Panels
+//TMentor
+/proc/is_mentor(client/C)
+	if(!istype(C))
+		return FALSE
+	if(!C.holder)
+		return FALSE
+	if(C.holder.rank.rights == R_MENTOR)

```

Comment:
Isn't rights a bitflag? This should be `if(C.holder.rank.rights & R_MENTOR)` unless your intent here is to check if the client is *only* a mentor.